library/core/src/sync/atomic.rs

```diff
@@ -464,6 +464,23 @@ impl AtomicBool {
     /// **Note:** This method is only available on platforms that support atomic
     /// operations on `u8`.
     ///
+    /// # Migrating to `compare_exchange` and `compare_exchange_weak`
+    ///
+    /// `compare_and_swap` is equivalent to `compare_exchange` with the following mapping for
+    /// memory orderings:
```

Comment:
It's not equivalent though, the return type is different...

See https://github.com/rust-lang/rust/issues/80486.